docs/projects.md

```diff
@@ -22,8 +22,8 @@
     {
         "name": "Multiplayer Tutorials",
         "url": "/multiplayer-tutorials",
-        "imageUrl": "/static/tutorials/pizza-party/pizza-party-img.png",
-        "largeImageUrl": "/static/tutorials/pizza-party/tmnt.gif"
+        "imageUrl": "/static/tutorials/pizza-party/playtime.png",
+        "largeImageUrl": "/static/tutorials/pizza-party/playtime.png"
```

Comment:
Is this just an automated change?

Reply:
Yes. Sometimes they just happen. I'm not sure what triggers them.

Reply:
iirc this file is generated on pxt serve going through and grabbing the first thing in each row, so will update whenever that changes